Makefile

```diff
@@ -27,6 +27,12 @@ ifeq ($(BUILDKIT_PROGRESS),)
 export BUILDKIT_PROGRESS := plain
 endif
 
+PREEMPT_RT_SUPPORT := ""
+ifeq ($(PREEMPT_RT),1)
+  PREEMPT_RT_SUPPORT := "--preempt-rt-support"
```

Comment:
Which values are allowed for this?
It seems like it can be one of three things: true, false, or "--preempt-rt-support"??
But the Dockerfile expects true or false?
Something is confusing to me.

Reply:
Which values are allowed for this?
It seems like it can be one of three things: true, false, or "--preempt-rt-support"??
But the Dockerfile expects true or false?
Something is confusing to me.

Reply:
Seems the github is indeed broken today, I experience the same two-times-comments.

Anyway, this "--preempt-rt-support" is an argument for the build-kernel-yaml.sh script, which sets the build dockerfile argument, the PREEMPT_RT_SUPPORT=true.

Reply:
The chain is the following
Makefile -> kernel-build-yml.sh -> build.yml -> Dockerfile

Reply:
It sure would be less confusing if in the Makefile this was called KERNEL_BUILD_YML_ARG or something that that.

Reply:
This is very subjective :) for example i clearly memorize variables much better if they consist of max 3 nouns, more than that for me is usually garbage which is difficult to remember, like "kernel build yml arg" or "build kernel yml arg", or other combinations. 

But on this particular matter i don't have any preferences, since this is very minor,  I will update the PR, no problem.